cabal-install/Distribution/Client/Config.hs

```diff
@@ -430,7 +431,8 @@ initialSavedConfig = do
       globalWorldFile    = toFlag worldFile
     },
     savedConfigureFlags  = mempty {
-      configProgramPathExtra = toNubList extraPath
+      configProgramPathExtra = toNubList extraPath,
+      configView             = toFlag "default"
```

Comment:
What will happen if the new cabal-install tries to pass this flag to an old Cabal `setup` executable?